app/grandchallenge/core/templates/grandchallenge/partials/userlinks.html

```diff
@@ -9,7 +9,7 @@
             <img class="mugshot"
                  src="{{ user.user_profile.get_mugshot_url }}"
                  alt="User Mugshot"
-                 style="height: 1.5em; vertical-align: middle;"/>
+                 style="height: 1.33em; vertical-align: -25%;"/>
```

Comment:
Does the -25% work well in all resolutions?

Reply:
I think so, this is to match the alignment from font-awesome.